### PR TITLE
Add module: swoole

### PR DESCRIPTION
**Pull request by fibis**
Add swoole module to php 8

### CODE REVIEW

File: Dockerfiles/mods/Dockerfile-8.0 [hunk 1]
```diff
@@ -537,6 +537,17 @@ RUN set -eux \
 	&& true
 
 
+# -------------------- Installing PHP Extension: swoole --------------------
```
Comment (cytopia) [inline Dockerfiles/mods/Dockerfile-8.0:540]: This is unfortunately not how it works. Dockerfiles are generated via Ansible from the configuration here. See section with swoole:

https://github.com/devilbox/docker-php-fpm/blob/master/build/ansible/group_vars/all/mods.yml#L989

**Review by cytopia — CHANGES_REQUESTED** [review]
Additionally, the README.md must also be updated accordingly (PHP extension added).

See Makefile targets:
```
make gen-dockerfiles
make gen-readme
```

**Comment by fibis** [comment]
@cytopia oh, that is sexier as I thought. But is it correct that  `make gen-readme` doesn't change anything?

**Comment by cytopia** [comment]
> @cytopia oh, that is sexier as I thought. But is it correct that `make gen-readme` doesn't change anything?

That will only update the README based on the docker images available locally. So either you need to builds the `mods` flavour locally, or just add the `swoole` extension for PHP 8.0 (see PHP 7.4, 7.3... where exactly it was added)

**Comment by fibis** [comment]
Thanks, I will work on that

**Pull request closed without merging**